docs/_config.yml

```diff
@@ -455,6 +455,9 @@ ar:
         - path: ar/week06/06-1.md
         - path: ar/week06/06-2.md
         - path: ar/week06/06-3.md
+    - path: ar/week07/07.md
```

Comment:
There is no such file.

Reply:
hi, i hope this PR #587 fix this !

Reply:
Sure, but you cannot include stuff that does not exist.